### DISCUSSION TITLE
search removed from primevue.org

### DISCUSSION THREAD

**Discussion by alqa-dash** [primefaces/community#2165] [PrimeVue]
Hey, recently discovered that search by Command-K silently disappear from primevue.org site. It was very convenient to search and switch among components. Is it possible to get it back?

**Comment by sfreeman-f5** [comment]
I also am sad the search is missing.

Also, the dark theming seems broken, I have to use darkreader browser extension which doesn't look as great.

**Comment by zach-mcnulty** [comment]
Plus 1 on this for the primevue v3 docs since v4 for released
